db/migrate/20230816074827_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.string :bio
+
```

Comment:
- [ ] Kindly remove the bio attribute from the post table because it is not on the ERD listed attributes, even if it was there, it is not necessary to have a bio in a post table.

Reply:
Thank you